### PR TITLE
refactor/164 lint issues

### LINKED ISSUE #144: Address various compiler warnings
There are various compiler warnings floating around from different merges which should be addressed

### LINKED ISSUE #164: Address Lint Issues

### PR DESCRIPTION
**Pull request by martin-lenweiter**
The warnings I suppressed can be seen in the app build.gradle file. The others have been fixed, except for one that will be fixed next week when making the UI compatible across different device formats.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -54,8 +44,11 @@
         - To set an image as representative of the category : click longer on the image and select the star icon -</string>
     <string name="Yes">Yes</string>
     <string name="No">No</string>
-    <string name="Close">Close</string>
-    <string name="numberOfSelectedPicturesText">%d selected pictures</string>
+    <plurals name="numberOfSelectedPicturesText">
+        <item quantity="zero">%d picture selected </item>
```
Comment (bbjubjub2494) [inline app/src/main/res/values/strings.xml]:
```suggestion
```
Unlike in French, in English grammar "zero pictures" is considered plural. [Source](https://en.wikipedia.org/wiki/Grammatical_number#:~:text=english%20treats%20zero%20with%20the%20plural%20number.)

**Review by wattsecs — CHANGES_REQUESTED** [review]
Maybe [suppress](https://developer.android.com/reference/java/lang/SuppressWarnings) the two warnings in LearningFragment because they do not really seem relevant

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -1,9 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <!-- Default screen margins, per the Android Design guidelines. -->
```
[the remark this answers is not in the document]
Comment (martin-lenweiter) [reply app/src/main/res/values/dimens.xml:3]: I think it got cleaned up automatically when I ran the IDE tool to remove unused resources, but okay I'll add them back.

[the remark this answers is not in the document]
Comment (mlenw) [reply app/src/main/res/values/dimens.xml:3]: I think it got cleaned up automatically when I ran the IDE tool to remove unused resources, but okay I'll add them back.

File: app/build.gradle [hunk 3]
```diff
@@ -181,10 +195,10 @@ dependencies {
 
     debugImplementation "androidx.fragment:fragment-testing:$fragment_version"
 
-    implementation "androidx.camera:camera-core:1.1.0-alpha02"
-    implementation "androidx.camera:camera-camera2:1.1.0-alpha02"
-    implementation "androidx.camera:camera-lifecycle:1.1.0-alpha02"
-    implementation "androidx.camera:camera-view:1.0.0-alpha22"
```
Comment (bbjubjub2494) [inline app/build.gradle:187]: I think last time I tried to do that bump it broke something, going to check real quick

**Review by wattsecs — APPROVED** [review]

File: app/build.gradle (hunk 3 above)
Comment (bbjubjub2494) [reply app/build.gradle:187]: nevermind

**Review by bbjubjub2494 — CHANGES_REQUESTED** [review]
removing fork_rep seems to break the utensils dataset on master. Please rebase

**Comment by bbjubjub2494** [comment]
0df1ff843c7f5d12ec39509b4bc886f15e5ecbbe

File: app/build.gradle [hunk 4]
```diff
@@ -83,6 +83,10 @@ android {
         // Didn't find a way to fix this, but it does not seem to cause any issue as we've been
         // using the learning for a while
         disable 'ClickableViewAccessibility'
+
+        // Most unused resources have been removed, but the removing the representative pictures
+        // seems to break the app, so these have been kept
+        disable 'UnusedResources'
```
Comment (bbjubjub2494) [inline app/build.gradle]: That's not necessary: it's not that it "seems to break the app" in some unexplained way, it's very much explained by [this](https://github.com/HumanLearning2021/HumanLearningApp/commit/0df1ff843c7f5d12ec39509b4bc886f15e5ecbbe#diff-71c4f73cc25e4063965a0465ecdc48d7307c911deed45f7a46192326bdbc9106R181). (To be fair my comment wasn't very clear)

Please remove this and rebase on main

**Review by bbjubjub2494 — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 4 above)
Comment (martin-lenweiter) [reply app/build.gradle]: i see thanks, done.

**Review by bbjubjub2494 — APPROVED** [review]
LGTM, maybe merge main before merging just to be sure

**Pull request merged**